tests/test_cookbook.py

```diff
@@ -234,7 +235,10 @@ def test_write_numeric_with_annotation_simpleion():
     # http://amzn.github.io/ion-docs/guides/cookbook.html#reading-numeric-types
     value = IonPyFloat.from_value(IonType.FLOAT, 123, (u'abc',))
     data = simpleion.dumps(value, binary=False)
-    assert u'$ion_1_0 abc::123.0e0' == data
+    if c_ext:
+        assert u'$ion_1_0 abc::123e+0' == data
+    else:
+        assert u'$ion_1_0 abc::123.0e0' == data
```

Comment:
Instead of checking whether the C extension is enabled, let's just make it `assert u'$ion_1_0 abc::123.0e0' == data or assert u'$ion_1_0 abc::123e+0' == data`